### PR TITLE
Feat: add encoder for identity service

### PR DESCRIPTION
**Pull request by Nhsdkk**
1) Add bcrypt to packages
2) Create Encoder class

### CODE REVIEW

File: identity_service/utils/encoder.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    """
+    Encoder class
+
+    Methods
+    _______
+    encode(password)
+        Returns hashed password
+    compare(password, hashed_password)
+        Returns True if passwords match and False if not
+    """
```
Comment (ZixMai) [inline identity_service/utils/encoder.py:16]: Add empty line

File: identity_service/utils/encoder.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+"""Encoder class"""
+from bcrypt import gensalt, hashpw, checkpw
+
+
+class Encoder:
+    """
+    Encoder class
+
+    Methods
+    _______
```
Comment (ZixMai) [inline identity_service/utils/encoder.py]: remove underscores and type "------" instead

File: identity_service/utils/encoder.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        password
+            Unhashed password
+        hashed_password
+            Hashed password
+
+        Returns
+        -------
+        bool
+            Flag if passwords are matching
+        """
```
Comment (ZixMai) [inline identity_service/utils/encoder.py:54]: empty string

**Review by ZixMai — CHANGES_REQUESTED** [review]

File: identity_service/utils/encoder.py (hunk 3 above)
Comment (Nhsdkk) [reply identity_service/utils/encoder.py:54]: fixed

File: identity_service/utils/encoder.py (hunk 2 above)
Comment (Nhsdkk) [reply identity_service/utils/encoder.py]: fixed

File: identity_service/utils/encoder.py (hunk 1 above)
Comment (Nhsdkk) [reply identity_service/utils/encoder.py:16]: fixed

**Review by ZixMai — APPROVED** [review]

**Review by MaxPlays35 — APPROVED** [review]

File: identity_service/pyproject.toml [hunk 4]
```diff
@@ -12,6 +12,7 @@ grpcio-tools = "^1.62.1"
 2to3 = "^1.0"
 prisma = "^0.13.0"
 redis = "^5.0.3"
+bcrypt = "^4.1.2"
```
Comment (MaxPlays35) [inline identity_service/pyproject.toml:15]: Maybe use this library. It has good, detailed documentation. https://pypi.org/project/pycryptodome/

**Review by MaxPlays35 — CHANGES_REQUESTED** [review]

**Review by MaxPlays35 — COMMENTED** [review]
See my comment

File: identity_service/pyproject.toml (hunk 4 above)
Comment (MaxPlays35) [inline identity_service/pyproject.toml:14]: Try this library. It has good, detailed documentation. https://pypi.org/project/pycryptodome/

Comment (Nhsdkk) [reply identity_service/pyproject.toml:14]: Add to issues, to resolve later

**Review by ZixMai — APPROVED** [review]

**Review by MaxPlays35 — APPROVED** [review]

**Pull request merged**